src/core/icons/fileIcons.ts

```diff
@@ -1601,6 +1601,8 @@ export const fileIcons: FileIcons = {
         '.eslintrc.base.json',
         '.eslintignore',
         '.eslintcache',
+        'eslint',
```

Comment:
Please remove the `eslint` as well. The pattern is defined below already:

<img width="298" height="63" alt="Image" src="https://github.com/user-attachments/assets/0feb96d8-de5a-4012-999a-5eb84325f160" />

See: https://github.com/material-extensions/vscode-material-icon-theme/pull/3156/files#diff-90ad265f1687cf6e1c278661d22de76f84d2ed788c6c0fb0694365567746f501R1607-R1609


```suggestion
```

Reply:
@PKief 
Thank you very much. Upon checking, I found that the icon theme built into vscode was being applied.
When I selected this extension, the icon in eslint.config.ts also changed to the eslint icon.
Therefore, I made the correction as you suggested.